### ISSUE TITLE
Templates does not support Linux arm64 etc.

### ISSUE THREAD

**Issue by eitsupi** [extendr/rextendr#241]
Reported in eitsupi/prqlr#88 by @andresrcs

It seems that packages based on rextendr templates such as `extendr/helloextendr` cannot be installed on Linux arm64 platform.
I wonder if rextendr need to generate Cargo.toml that would enable  libR-sys's `use-bindgen` feature except on certain platforms?
https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html#platform-specific-dependencies

**Comment by eitsupi** [comment]
> Another option is to fix this on libR-sys's side. The CI should be able to generate the bindings for `aarch64-unknown-linux-gnu` by cross-compiling.

Thanks for the reply. This is definitely an attractive option and I hope this will happen at least on linux/arm64.

However, I still think the template needs to be improved, since it will not be possible to install any packages generated from rextendr on other platforms.

**Comment by eitsupi** [comment]
I have tried working as advised by @yutannihilation, but it doesn't seem to work.
It would be helpful if you could take a look at extendr/libR-sys#133.

**Comment by CGMossa** [comment]
Oh sorry I didn't approve the run on CI.
Ping me if something simple has to happen, but it hasn't been done.

**Comment by eitsupi** [comment]
@CGMossa Thank you for looking at this.
I created extendr/helloextendr#17 simply to see if extendr/libR-sys#133 would work on a linux/arm64 machine (via `remotes::install_github("extendr/helloextendr#17")`), so CI does not need to be running.

**Comment by CGMossa** [comment]
Good. I thought you wanted to use the PR and CI as a means to test the
target.
Good work either way.

**Comment by JosiahParry** [comment]
Is this fixed? I work exclusively on arm64 and have never ran into an issue.

**Comment by Ilia-Kosenkov** [comment]
No, not yet. At least we made no attempts to fix that. I am unsure if there is an issue still.

**Comment by eitsupi** [comment]
> Is this fixed? I work exclusively on arm64 and have never ran into an issue.

See extendr/libR-sys#133.
Linux arm64 is now supported since bindings were added to libR-sys, but continues to be not supported on all platforms except Windows x86, Windows amd64, macOS amd64, macOS arm64, Linux amd64, and Linux arm64.
